BiweeklyMeetings/2022-08-16.md

```diff
@@ -0,0 +1,406 @@
+
+**Robert Phair** -: Michael, this is a time one that I got you to chime in on the forum about, you made a comment there. So I think it is one of the ones that you've seen.
+
+**Michael** -: The notes. I was looking at a couple of those and there was that I believe it's CIP 37, this one, 45 and I guess 50, that is all with pledge and the centralization metrics is concerned. It's very, well, it's hard to go through you, you'd expect so. [inaudible 00:07:03] so I don't know, I wish I could give us [inaudible 00:07:04] and maybe there's a way that we can bundle them, simpler path to a review. I don't remember it's particularities on the phone.
+
+**Matthias** -: The way I see it currently is, we have CIP 50, where most of the [inaudible 00:07:22] discussion really happens, which is probably the most structured of all those proposals that we have at least in terms of the [inaudible 00:07:30] operations, if I can say, or how the feedback is incorporated in the CIP. So all those other CIPs that are kind of overlapping, but not exactly the same, are they think the same issues as CIP 50, I think they should be part of the same discussions because they all have a similar set of assumptions. They all have a similar set of the outcome. The question is now, which one are we going to adopt in the end? So if there is a more generic discussion happening on this decentralization concern, we want an incentive on [inaudible 00:08:07]. I think it's best driven from CIP 50 and includes feedback or update from all those other site proposals that happened.
+
+**Matthias** -: I think this is a bit what Michael is also saying in the chats, by saying I've included CIP 45 in the CIP 50 references since the first draft. And I think it's good to keep that as part of this, the discussions, because in a way it's part of the whole same discussions. And as we are now trying to get more and more attention from IUG research, I mean, we've got not going to have that discussion five time, one for each of those CIPS, but probably once and for all, at least for the next proposal to come and see which direction we take.
+
+**Robert Phair** -: I've also suggested to the CIP 45 author that he come into the discord on the CIP 50 channel, and to review what's already happened and to explain to others that have been following the issue for their issue, how his presentation is different and similar.
+
+**Matthias** -: Yeah, so perhaps that's also kind of, I don't know, part of CIP 50 rational section, they have a kind of alternatives considered and list a bit of the pro and cons of those various other CIPs and why maybe what was taken from them, what was rejected from them for which reason, because it might actually also have discussions and having one place to centralize all the results there might help too.
```

Comment:
```suggestion
**Matthias** -: Yeah, so perhaps that's also kind of, I don't know, part of CIP 50 rationale section, they have a kind of alternatives considered and list a bit of the pro and cons of those various other CIPs and why maybe what was taken from them, what was rejected from them for which reason, because it might actually also have discussions and having one place to centralize all the results there might help too.
```